src/components/ViewOptionsPanel.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="panel">
+    <p class="panel-heading" @click.stop="isOpen = ! isOpen">
+      <span>Ansichtseinstellungen</span>
+      <span class="icon is-medium clickAble right">
+        <font-awesome-icon v-if="isOpen" icon="chevron-up" />
+        <font-awesome-icon v-else icon="chevron-down" size="1x" />
+      </span>
+    </p>
+    <form class="panel-block form">
+      <div class="field is-horizontal">
+        <div class="field-body">
+          <div id="view-settings" class="field" v-if="isOpen">
+            <div class="control">
+              <div class="buttons">
+              <a class="button" @click="togglePseudonyms">
```

Comment:
meine Editor/Compiler ist hier und in den nächsten Zeilen mit den Einrückungen nicht einverstanden ;-)

Vorschlag: `npm run lint` ausführen 
Das muss nicht sofort sein und aber zB vor dem Commit, spätestens vor dem pull request